Makefile.common

```diff
@@ -267,3 +267,9 @@ $(1)_precheck:
 		exit 1; \
 	fi
 endef
+
+govulncheck: install-govulncheck
+	govulncheck ./...
+
+install-govulncheck:
+	which govulncheck || go install golang.org/x/vuln/cmd/govulncheck@latest
```

Comment:
prefer `command -v govulncheck > /dev/null` over `which govulncheck`
See https://unix.stackexchange.com/a/85250/67917

Reply:
Thank you for the review. I've updated my changes as per the recommendation.